src/network.ts

```diff
@@ -0,0 +1,25 @@
+import { TransactionVersion, ChainID } from './constants';
+
+export interface StacksNetwork {
```

Comment:
This is great API design, nice work @yknl

Reply:
💯